crucible/src/Lang/Crucible/CFG/SSAConversion.hs

```diff
@@ -126,24 +131,24 @@ data BlockInput ext s blocks ret args
              -- | Arguments expected by block.
            , binputArgs       :: !(Assignment (Value s) args)
            , binputStmts      :: !(Seq (Posd (Stmt ext s)))
-           , binputTerm       :: !(Posd (TermStmt s ret))
+           , binputTerm       :: !(Posd (Either (JumpInfo s blocks) (TermStmt s ret)))
```

Comment:
This `Either` is pretty inscrutible... let's define a new datatype instead with more descriptive constructor names.

Reply:
👍